backend/Dockerfile

```diff
@@ -87,8 +87,11 @@ ENV TZ=UTC \
     MAX_FILE_SIZE=100000000 \
     USE_ENCRYPTION=false \
     # Use to enable cloud specific feature
-    CLOUD="aws"
-
+    CLOUD="aws" \
+    # Use to set compression threshold for tracker requests (20kb by default)
+    COMPRESSION_THRESHOLD="20000" \
```

Comment:
Please, change it to int value (without "") as other integer variables in Dockerfile